app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/manage/actions.ts

```diff
@@ -66,3 +69,17 @@ export const resetThrottlingRate = AuthenticatedAction(async (session, formId: s
     return { error: "There was an error. Please try again later." } as ServerActionError;
   }
 });
+
+export const updateNotificationsInterval = AuthenticatedAction(
+  async (session, formId: string, notificationsInterval: NotificationsInterval) => {
```

Comment:
`session` can be changed to `_` if the variable will be unused later in the code.  This will quiet the linter as well.